external/netty/dockerfile/netty-test.sh

```diff
@@ -0,0 +1,48 @@
+#/bin/bash
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#Set up Java to be used by the the example-test
+
+if [ -d /java/jre/bin ];then
+	echo "Using mounted Java8"
+	export JAVA_BIN=/java/jre/bin
+	export JAVA_HOME=/java
+	export PATH=$JAVA_BIN:$PATH
+elif [ -d /java/bin ]; then
+	echo "Using mounted Java9"
+	export JAVA_BIN=/java/bin
+	export JAVA_HOME=/java
+	export PATH=$JAVA_BIN:$PATH
+else
+	echo "Using docker image default Java"
+	java_path=$(type -p java)
+	suffix="/java"
+	java_root=${java_path%$suffix}
+	export JAVA_BIN="$java_root"
+	echo "JAVA_BIN is: $JAVA_BIN"
+	export JAVA_HOME="${java_root%/bin}"
+fi
+
+TEST_SUITE=$1
+java -version
+
+# Replace the following with the initial command lines that trigger execution of your test
+# For this example, we will simply compile and run the ExampleTest class
+pwd
+
+set -e
+cd /netty/
+mvn clean package
+echo "Compile and execute example-test" && \
```

Comment:
Can update to echo netty-test rather than example-test

Reply:
Moving this comment to the line before `mvn clean package` would be clearer?